aws-eks-addon/variables.tf

```
@@ -0,0 +1,6 @@
+variable "eks_properties" {}
+variable "addon_properties" { type = object({
+        coredns_version = number
+        kube_proxy_version = number
+        vpc_cni_version = number
+})}
```

you can change addon properties to 

```suggestion
variable "addon_properties" { type = map(string) }
```

the keys will be the addon name and the values will be the addon version

---

fixed